internal/config/config.go

```diff
@@ -224,6 +224,8 @@ var (
 	AssetManagerRetryMaxDelay = rootKey("asset.manager.retry.maxDelay")
 	// AssetManagerRetryFactor the backoff factor to use for retry of database operations
 	AssetManagerRetryFactor = rootKey("asset.manager.retry.factor")
+	// AssetManagerResolveKeysViaBlockchainPlugin whether to verify/resolve keys via the blockchain plugin (supporing facilities like key aliases), but requiring the same backing blockchain connector
+	AssetManagerResolveKeysViaBlockchainPlugin = rootKey("asset.manager.resolveKeysViaBlockchainPlugin")
```

Comment:
If this is false, does that mean keys are not resolved at all? Or there is some "other" way to resolve keys?

I'm probably just trying to turn a `bool` into an `enum` again :) just trying to be sure the implications of true vs false are clear here.

Reply:
I see you explained [here](https://github.com/hyperledger/firefly/pull/549#discussion_r814465236), so sounds like:
* `resolveKeysViaBlockchainPlugin=true` means keys passed to asset/token routes will be validated via the blockchain plugin
* `resolveKeysViaBlockchainPlugin=false` means keys passed to asset/token routes will not be validated at all (which was actually the preexisting behavior)

I guess just wondering how this might be spelled if it's more extensible down the road...

Reply:
I shall have a think and make a proposal. Overall I'd like to find a better word than `resolve` for the thing we do here, but one hasn't sprung to mind yet.

Reply:
... considering `Normalize` unless something better springs to mind. Then it can be `keyNormalization: "blockchain"|"none"|...tbd`

Reply:
Works for me

Reply:
ok - that's done